core/src/main/scala/kafka/server/DynamicConfigManager.scala

```diff
@@ -143,43 +106,20 @@ class DynamicConfigManager(private val zkUtils: ZkUtils,
           configHandlers(entityType).processConfigChanges(entity, entityConfig)
 
         case o => throw new IllegalArgumentException("Config change notification has an unexpected value. The format is:" +
-                                                             "{\"version\" : 1," +
-                                                             " \"entity_type\":\"topic/client\"," +
-                                                             " \"entity_name\" : \"topic_name/client_id\"}." +
-                                                             " Received: " + json)
-      }
-    }
-  }
-
-  private def purgeObsoleteNotifications(now: Long, notifications: Seq[String]) {
```

Comment:
Don't you need to do this purging any longer? I'm just confirming that it is ok to get rid of this code.